### PR TITLE
add subcriteria base template

### PR DESCRIPTION
**Pull request by Adam-W1**
https://digital.dclg.gov.uk/jira/browse/FS-2029

### CODE REVIEW

**Review by tferns — APPROVED** [review]
Nice work on this, functionally I think it looks good - have left some comments on the variables we're passing to the templates though, could maybe remove some redundancies

File: app/assess/routes.py [hunk 1]
```diff
@@ -73,54 +68,63 @@ def application_sub_crit_scoring(application_id: str, sub_criteria_id: str):
...
+                fund=fund,
+                form=form,
+                workflow_status=sub_criteria.workflow_status,
+                scores_submitted=scores_submitted,
+                score_list=score_list if len(score_list) > 0 else None,
+                latest_score=latest_score,
+                COF_score_list=COF_score_list,
+                score_error=score_error,
+                justification_error=justification_error,
+                project_reference=sub_criteria.project_reference
```
[the remark this answers is not in the document]
Comment (adamdavies1) [reply app/assess/routes.py]: yup good shout, have changed this now

File: app/assess/routes.py [hunk 2]
```diff
@@ -73,54 +68,63 @@ def application_sub_crit_scoring(application_id: str, sub_criteria_id: str):
...
-    on_summary = True if theme_id == "score" else False
+    
+    args = request.args
+    theme_id = args["theme_id"]
+    sub_criteria = get_sub_criteria(application_id=application_id, sub_criteria_id=sub_criteria_id)
+    fund = get_fund(Config.COF_FUND_ID)
 
-    sub_criteria = get_sub_criteria(sub_criteria_id=sub_criteria_id)
+    if theme_id == "score":
+        on_score = True
```
[the remark this answers is not in the document]
Comment (adamdavies1) [reply app/assess/routes.py]: removed this variable now

File: app/assess/templates/macros/sub_criteria_navbar.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{% macro navbar(application_id, sub_criteria, themes, current_theme_id, on_summary) %}
```
[the remark this answers is not in the document]
Comment (adamdavies1) [reply app/assess/templates/macros/sub_criteria_navbar.html]: Yup

File: app/assess/templates/macros/sub_criteria_navbar.html [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+<nav class="assessment-navigation">
+    <ul class="govuk-list">
+        {%for theme in themes%}
```
[the remark this answers is not in the document]
Comment (adamdavies1) [reply app/assess/templates/macros/sub_criteria_navbar.html]: and yup

File: app/assess/templates/sub_criteria.html [hunk 5]
```diff
@@ -1,105 +1,66 @@
...
-                <div class="govuk-grid-column-one-third">
-                    <a href="#" class="govuk-back-link">Back to <span class="govuk-!-font-weight-bold">assessment
-                            overview</span></a>
-                    <h1 class="govuk-heading-l scoring-heading">Score engagement</h1>
-
-                </div>
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+            <div class="govuk-grid-column-one-third">
+                {{ navbar(application_id,sub_criteria,sub_criteria.themes, current_theme_id, on_summary) }}
```
[the remark this answers is not in the document]
Comment (adamdavies1) [reply app/assess/templates/sub_criteria.html]: yup again

**Pull request merged**